sdk/wallet-adapter/README.md

```diff
@@ -92,6 +92,7 @@ The `WalletStandardAdapterProvider` adapter (published under `@mysten/wallet-ada
 The following wallets are known to work with the Wallet Standard:
 
 - **[Sui Wallet](https://docs.sui.io/devnet/explore/wallet-browser)**
+- **[Suiet Wallet](https://suiet.app/)**
```

Comment:
Can you keep this alphabetically sorted, with the Sui wallet on top?

Reply:
Oh I got what u mean. You mean Sui Wallet on top and the rest items follow alphabet order? I changed it now.
But it looks strange that our 'S'uiet doesn't follow after 'S'ui